cmd/kubectl-gadget/trace/tcpconnect.go

```diff
@@ -15,113 +15,29 @@
 package trace
 
 import (
-	"fmt"
-	"strings"
-
 	commontrace "github.com/kinvolk/inspektor-gadget/cmd/common/trace"
-	commonutils "github.com/kinvolk/inspektor-gadget/cmd/common/utils"
 	"github.com/kinvolk/inspektor-gadget/cmd/kubectl-gadget/utils"
 	"github.com/kinvolk/inspektor-gadget/pkg/gadgets/trace/tcpconnect/types"
 
 	"github.com/spf13/cobra"
```

Comment:
Could you please take the opportunity to reorganize the order here and follow the new notation we are trying to adopt: 

```suggestion
	"github.com/spf13/cobra"

	commontrace "github.com/kinvolk/inspektor-gadget/cmd/common/trace"
	"github.com/kinvolk/inspektor-gadget/cmd/kubectl-gadget/utils"
	"github.com/kinvolk/inspektor-gadget/pkg/gadgets/trace/tcpconnect/types"
```